tutorials/models/1_gnn/4_rgcn.py

```diff
@@ -361,5 +361,4 @@ def forward(self, g):
 #
 # The implementation is similar to that presented here, but with an extra DistMult layer
 # stacked on top of the R-GCN layers. You can find the complete
-# implementation of link prediction with R-GCN in our `Github Python code example
-#  <https://github.com/dmlc/dgl/blob/master/examples/pytorch/rgcn/link_predict.py>`_.
+# implementation of link prediction with R-GCN in our `Github Python code example <https://github.com/dmlc/dgl/blob/master/examples/pytorch/rgcn/link.py>`_.

```

Comment:
this line is too long. could you just split into 2 lines to keep them short?

Reply:
Changed it.